Commands/Search/GetSiteHistoricalVersionsStatus.cs

```diff
@@ -0,0 +1,56 @@
+﻿#if !ONPREMISES
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.SharePoint.Client;
+using Microsoft.SharePoint.Client.Search.Administration;
+using PnP.PowerShell.CmdletHelpAttributes;
+
+namespace PnP.PowerShell.Commands.Search
+{
+    [Cmdlet(VerbsCommon.Get, "PnPSiteHistoricalVersionsStatus", DefaultParameterSetName = "Xml")]
+    [CmdletHelp("Returns information about the Historical Versions feature for the current site collection from the context. " +
```

Comment:
Can you better explain what "Historical Versions feature" is? What does it return in terms of data, and what would you use it for.

Reply:
Updated description with a bit more info